### ISSUE TITLE
Unable to open quest from a Wanted Poster (BB #912)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#912]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 26.05.2015 18:49:58 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/912

<hr>

Unable to open quest from that Wanted Poster in Terokkar forest (Stonebreaker Hold)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 27.05.2015 04:11:03 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/912/#comment-18379203

<hr>

Seems to be a problem with gameobject faction 102

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 28.05.2015 18:36:53 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/912/#comment-18436276

<hr>

Could this be related to issue about hive tower

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 28.05.2015 19:42:37 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/912/#comment-18437453

<hr>

Fix #898, #912 & #909

→ <<cset 662f81fc79f4>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.05.2015 01:22:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/912/#comment-18473903

<hr>

[Core/Opcodes] Fix a sanity check in HandleGameObjectUseOpcode that was added to
               block certain exploits. As far as I'm aware, the fix is still valid
               but now we have returned a majority of broken gameobjects to normal.

```
           Fixes #931, #943.
           Correctly fixes #912, #909 and #898.
```

→ <<cset e215d2efab6f>>
